crates/ark/src/lsp/completions/completion_item.rs

```diff
@@ -175,8 +176,9 @@ pub(super) fn completion_item_from_function<T: AsRef<str>>(
 
     item.kind = Some(CompletionItemKind::FUNCTION);
 
-    let detail = format!("{}({})", name, parameters.joined(", "));
-    item.detail = Some(detail);
+    let detail = format!("({})", parameters.joined(", "));
+    let label_details = item_details(Some(detail), package);
+    item.label_details = Some(label_details);
```

Comment:
I agree that we have many ways to see the function parameters, and by far the most useless one is seeing the function parameters in this completion item menu - the parameters are very scrunched together and you can't see their defaults and they are often cut off

I vote for just showing the package name, and doing a little work to strip out `parameters` here all together, since its probably a non trivial amount of work to compute it and we won't need it anymore.

Reply:
Not sure if it is relevant but does this also apply for Python package?

Reply:
No, completions for Python are not generated from ark; ark is a Jupyter kernel for R.